OxfordBodleian/Juel-Jensen/BDLaethb3.xml

```diff
@@ -10,18 +10,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Four Gospels</title>
-                <!-- 
-                        <title xml:lang="gez" xml:id="title1">አርባዕቱ፡ ወንጌል፡</title>
+                <title xml:lang="gez" xml:id="title1">አርባዕቱ፡ ወንጌል፡</title>
                 <title xml:lang="gez" corresp="#title1" type="normalized">ʾArbāʿəttu wangel</title>
```

Comment:
ʾArbāʿtu, no?

Reply:
That's a good question. Ethio-SPaRe and the work record LIT1560Gospel have ʾArbāʿtu, the EAE has ʾArbāʿətu. The geminated form ʾArbāʿəttu is given by Wolf Leslau, taking from the traditional pronunciation. I am afraid there has never been a discussion on the spelling of this and other words. 
Since BM has ʾArbāʿtu, perhaps it is better to use the same spelling here too.

Reply:
It is on the list inherited from TraCES https://betamasaheft.eu/Guidelines/?id=dubious-spelling (which is not necessarily binding for us, so as long as there are no strong arguments against it, let's go with ʾArbāʿtu)